src/common/color/helper.ts

```diff
@@ -1,9 +1,11 @@
-import { scaleOrdinal } from 'd3-scale';
-import { maxIndex } from 'd3-array';
+import { scaleOrdinal, scaleQuantile } from 'd3-scale';
+import { maxIndex, extent } from 'd3-array';
 import { schemes } from './schemes';
+import { uniqueBy } from '../utils';
 
 export type ColorSchemeType =
   | ((data, index: number, active?: any[]) => string)
+  | Partial<CSSStyleDeclaration>[]
```

Comment:
So this new shape for a colorScheme will only work for heatmaps at the moment. Maybe I should pull it out into it's own type and then apply it as a union type to only heatmaps for now

```
type ColorSchemeStyleArray = Partial<CSSStyleDeclaration>[];

// HeatmapSeries.tsx
...
colorScheme: ColorSchemeType | ColorSchemeStyleArray;

```

Reply:
Ya lets make this a uniion